gradle.properties-example

```diff
@@ -26,6 +26,7 @@ wp.zendesk.domain=https://www.google.com/
 wp.zendesk.oauth_client_id=wordpress
 wp.reset_db_on_downgrade = false
 wp.sentry.dsn=https://00000000000000000000000000000000@sentry.io/00000000
+jp.sentry.dsn=https://00000000000000000000000000000000@sentry.io/00000000
```

Comment:
Just a note / reminder for myself that this will require an update of the `gradle.properties` in mobile secrets then a bump of the hash in `.configure` once we agree that this is the right idea.